modules/Makefile.kubernetes

```diff
@@ -20,11 +20,13 @@ export SERIAL
 
 define kubectl_create
 	@echo -e "INFO: Creating $(KUBERNETES_APP) $(1) on cluster $(call yellow,$(CLUSTER_NAMESPACE))..."
+	@[ -z "$(DEBUG)" ] && (envsubst < "$(KUBERNETES_RESOURCE_PATH)/$(KUBERNETES_APP)-$(1).yml" | envsubst)
```

Comment:
Its slightly annoying that this means if you want to make changes you need to do it in two places. 

What if you instead made `envsubst` a variable and it dynamically added the `@` if `DEBUG` was false? Then you wouldnt need the duplicate line, and you would get the full output of the kubectl line